src/app/campaign/inventory/inventory-table.component.ts

```diff
@@ -8,8 +8,17 @@ import { getMidnightUTC, getDateSlice } from '../store/selectors';
     <div class="row head">
       <div class="expand"></div>
       <div class="date">Week</div>
-      <div class="avail">Available</div>
-      <div class="goal">Allocated</div>
+
+      <ng-template [ngIf]="uncapped" [ngIfElse]="capped">
```

Comment:
Ugh, looked into this, and there is definitely a problem with template re-rendering here, but it doesn't seem _entirely_ like our bug. It didn't seem like component would re-render just by setting `comp.flight.uncapped = true` because the Object reference does not change by setting a property, and change detection does not run on deep object property changes. So I changed that to `comp.flight = {..comp.flight, uncapped: true}`, nope still seeing the Allocated heading.  Then I found this https://github.com/angular/angular/issues/12313 and tried adding a ParentComponent wrapper in the test, no luck. I also give up on this test.

You do have one problem though, you're re-using the #capped template ref, so in my test I was actually see the numeric values in the second #capped template ref. Those need to be unique.

Reply:
Good to know I'm not crazy.  Or that we're both equally crazy.

Will fix those refs!